frontend/packages/policy-editor/src/utils/ExpandablePolicyCardUtils/index.ts

```diff
@@ -52,8 +52,8 @@ export const getSubjectOptions = (subjects: PolicySubject[], policyRule: PolicyR
  */
 export const getActionOptions = (actions: PolicyAction[], policyRule: PolicyRuleCard) => {
   return actions
-    .filter((a) => !policyRule.actions.includes(a.actionTitle))
-    .map((a) => ({ value: a.actionTitle, label: a.actionTitle }));
+    .filter((a) => !policyRule.actions.includes(a.actionId))
+    .map((a) => ({ value: a.actionId, label: a.actionId }));
```

Comment:
How about handling the translation directly here instead of replicating it every time `getActionOptions` is utilized? 😊 

```suggestion
    .map((a) => ({ value: a.actionId, label:  getTranslationByActionId(action.id) }));
```

Reply:
I'm sure that is possible, but then we also need to pass in the translation function as this file is outside of the component.